src/content/docs/logs/ui-data/parsing.mdx

```diff
@@ -527,6 +537,131 @@ Note that variable names must be explicitly set and be lowercase like `%{URI:uri
     * <DNT>**region**</DNT>: Abbreviation of state, province, or territory
     * <DNT>**regionName**</DNT>: Name of state, province, or territory
   </Collapser>
+
+  <Collapser
+    id="parsing-key-value-pairs"
+    title="Parsing Key Value Pairs"
+  >
+    The New Relic logs pipeline parses your log messages by default, but sometimes you have log messages that are formatted as key-value pairs. In this situation, you may want to be able to parse them and then be able to filter using the key-value attributes.
+
+    If that is the case, you can use the `key value pairs` [grok type](#grok-syntax), which will parse the key-value pairs captured by the grok pattern. This format relies on 3 main parts: the grok syntax, the prefix you would like to assign to the parsed key-value attributes, and the `key value pairs` [grok type](#grok-syntax). Using the `key value pairs` [grok type](#grok-syntax), you can extract and parse key-value pairs from logs that are not properly formatted; for example, if your logs are prefixed with a date/time string:
```

Comment:
> you can extract and parse key-value pairs from logs that are not properly formatted; for example, if your logs are prefixed with a date/time string

Nit: no me, it's properly formatted, it's just that the entire message isn't kvp. So maybe this should say something like "you can extract and parse key-value pairs in the entire log or a specific portion of the log"